assets/queries/cloudFormation/aws/elasticache_nodes_not_created_across_multi_az/metadata.json

```
@@ -3,7 +3,7 @@
   "queryName": "ElastiCache Nodes Not Created Across Multi AZ",
   "severity": "MEDIUM",
   "category": "Availability",
-  "descriptionText": "ElastiCache Nodes should have 'AZMode' set to 'cross-az' in in multi nodes cluster",
+  "descriptionText": "ElastiCache Nodes should be created across multi az, which means 'AZMode' should be set to 'cross-az' in in multi nodes cluster",
```

Can you delete one of the double "in", please? (The same applies for `assets/queries/terraform/aws/elasticache_nodes_not_created_across_multi_az` and `assets/queries/pulumi/aws/elasticache_nodes_not_created_across_multi_az`)

```suggestion
  "descriptionText": "ElastiCache Nodes should be created across multi az, which means 'AZMode' should be set to 'cross-az' in multi nodes cluster",
```